qemu/tests/cfg/block_boot_multi_disks.cfg

```diff
@@ -0,0 +1,44 @@
+- block_boot_multi_disks:
+    type = block_boot_multi_disks
+    start_vm = no
+    vt_ulimit_nofile = 65536
+    only i386, x86_64
```

Comment:
After distinguishing the check_message parameter, this case can run normally on arm, so please add aarch64.